soh/src/code/z_sram.c

```diff
@@ -196,6 +197,19 @@ void Sram_OpenSave() {
         }
     }
 
+    // Setup the entrance override table for rando
+    if (gSaveContext.n64ddFlag) {
```

Comment:
Here, and the two below are the main uses of the "entrance override" logic, and relates to the early conversations above with @garrettjoecox. It achieves two things:
1. Allows us to modify the games entrance table for entrance skips or modifications (like skip zelda stealth)
2. Setups up the entrance shuffle table (uses vanilla data if un-shuffled)

The thought is point #1 could be useful outside of entrance shuffle, and since the shuffle table is initialized with vanilla data, it should be safe to use.

Lets decide what we want to with this PR. We could add the entrance shuffle check here for now, and re-visit it later when we figure out how we want to take advantage of a modified entrance table.

Reply:
as long as initializing the table doesn't break anything in non-entrance rando i say let's keep it, it'll be nice to use it to clean up some of the other entrance override stuff we're doing in hacky ways in the future